docs/core/get-started/sdk.md

```diff
@@ -49,5 +53,5 @@ The Ruby SDK is available [via Rubygems](https://rubygems.org/gems/chain-sdk). R
 To install, add the following to your `Gemfile`:
 
 ```
-gem 'chain-sdk', '~> 1.1.0', require: 'chain'
```

Comment:
The pesimistic version string `'~> 1.1.0'` is equivalent to `'>= 1.1.0', '< 1.2.0'` per  http://guides.rubygems.org/patterns/ when the "patch" version is specified. No change needed here.